API.md

```diff
@@ -0,0 +1,72 @@
+- `deviceCount`: the number of SegM8 modules connected in a daisy-chain. If omitted, defines a single module.
+
+### `SegM8(uint8_t pinCS, uint8_t pinDI, uint8_t pinCLK, uint8_t deviceCount = 1)`
+
+Constructs a new SegM8 object that uses a software SPI bus.
+
+- `pinCS`: the *chip select* (also known as *slave select*) pin is used to control the shift-register latch. It is marked **CS** on the SegM8 module. Any GPIO pin of the controller may be used.
+- `pinDI`: the *data input* pin used to send data to the peripherals. It is marked **DI** on the SegM8 module. In SPI terms known as **MOSI** (Master Out Slave In). Any GPIO pin of the controller may be used.
+- `pinCLK`: the *clock* pin is used to clock pulses that synchronize data transmission generated by the master. It is marked **CLK** on the SegM8 module. Also known as **SCK** in SPI terms. Any GPIO pin of the controller may be used.
+- `deviceCount`: the number of SegM8 modules connected in a daisy-chain. If omitted, defines a single module.
+
+The difference between hardware and software SPI is interaction speed. Software SPI runs slower than hardware SPI but hardware SPI works only on specific board pins: **MOSI**, **MISO** (not used in SegM8) and **SCK**.
+
+### `void begin()`
+
+Initializes the given interface, clear internal buffer, and sets all module segments to the «dark» state.
+Call this method before interacting with SegM8. For example, in the `setup()`.
+
+### `void clear()`
+
+Sets all module segments to the «dark» state. Clear internal buffer.
+
+### `void display(int number, uint8_t position, uint8_t width, uint8_t flags = SEGM8_ALIGN_RIGHT)`
+
+### `void display(unsigned int number, uint8_t position, uint8_t width, uint8_t flags = SEGM8_ALIGN_RIGHT | SEGM8_RADIX_10)`
+
+### `void display(long number, uint8_t position, uint8_t width, uint8_t flags = SEGM8_ALIGN_RIGHT)`
+
+### `void display(unsigned long number, uint8_t position, uint8_t width, uint8_t flags = SEGM8_ALIGN_RIGHT | SEGM8_RADIX_10)`
+
+### `void display(float number, uint8_t position, uint8_t width, uint8_t precission = 1, uint8_t flags = SEGM8_ALIGN_LEFT)`
+
+### `void display(const char* string, uint8_t position, uint8_t width, uint8_t flags = SEGM8_ALIGN_LEFT)`
+
+### `void display(String string, uint8_t position, uint8_t width, uint8_t flags = SEGM8_ALIGN_LEFT)`
+
+Prints fixed `width` number or text line to the `position` of output buffer, using formatting `flags`. Moves the contents of the buffer to the indicator daisy-chain Note: the `.` sign does not occupy a separate module at the output, but is displayed in the previous module.
+
+When printing text strings, the interpretation of characters in indicator segments is used; for unsupported characters **K, M, N, V, W, X, Z**, an `underscore` is displayed (segment **a**).
```

Comment:
```suggestion
When printing text strings, the interpretation of characters in indicator segments is used. Instead of unsupported characters (e.g. **K, M, N, V, W, X, Z**) an `underscore` is displayed (segment **a**).
```